### PR TITLE
feat: Handle connectors specified in ui-rdf

### PR DESCRIPTION
**Pull request by toftalisk**
We can now specifiy connectors using ui:hasConnector. The id specified
by this predicate need to match ids in svg-objects containing actual svg
+ these connectors. If we do this, we can connect edges to connectors instead
of actual svg-node. These connectors are small fake nodes that follow
its parent (svg node) using relative coordinates while themself are
unselectable and unaffected by layouts.

For now we have mocked the svg lib specifying SVGs with connectors and their
coordinates.

### CODE REVIEW

**Comment by toftalisk** [comment]
Waiting for @lorentzfb svg-lib

File: src/mapper/rdfTriples2Elements.ts [hunk 1]
```diff
@@ -2,69 +2,155 @@ import { ElementDefinition } from 'cytoscape';
...
+				id: connectorId,
+				parent: connector2IconNode[connectorId],
+				nodeType: NodeType.SymbolConnector,
+				layoutIgnore: true,
+			},
+			position: connectorData.point,
+			grabbable: false,
+			selectable: false,
+		};
+		console.log('Connector', connectorNode);
```
Comment (toftalisk) [inline src/mapper/rdfTriples2Elements.ts:151]:
```suggestion
```

File: src/mapper/rdfTriples2Elements.ts [hunk 2]
```diff
@@ -2,69 +2,154 @@ import { ElementDefinition } from 'cytoscape';
...
+					},
+				};
+
+				cyNode.data.layoutIgnore = false;
+				cyNode.data.nodeType = NodeType.SymbolContainer;
+				//cyNode.data.imageHeight = `${symbol.height}px`;
+				//cyNode.data.imageWidth = `${symbol.width}px`;
+				cyNode.data.rotation = 0;
+
+				console.log('Got image node ', cyNode);
```
Comment (toftalisk) [inline src/mapper/rdfTriples2Elements.ts:109]:
```suggestion
```

**Review by lorentzfb — APPROVED** [review]
🥇

**Pull request merged**